content/blog/2018/12/2018-12-14-java11-preview-availability.adoc

```diff
@@ -0,0 +1,237 @@
+---
+layout: post
+title: "Java 11 Support Preview availability in Jenkins 2.155+"
+tags:
+- core
+- developer
+- java11
+- community
+- platform-sig
+author: oleg_nenashev
+---
+
+NOTE: This is a joint blogpost prepared by the link:https://github.com/orgs/jenkinsci/teams/java11-support[Java 11 Support Team]
+
+image:/images/logos/formal_java/256.png[Jenkins Java, role=center, float=right]
+
+Jenkins, one of the leading open-source automation servers,
+still supports only Java 8 even though Java 9 and above have been officially released for some time.
+Over the last year Jenkins contributors have been working towards enabling support for Java 11 in the project (Jenkins JEP-211).
+It was a thorny path,
+but now, on behalf of the link:/sigs/platform[Jenkins Platform SIG],
+we are happy to announce preview availability of Java 11 support in Jenkins weekly releases.
+
+Java 11 preview availability offers Jenkins contributors and early adopters a way
+to try out the changes before the general availability release happens early next year.
+It should help us to get more exploratory testing and, hopefully,
+resolve the most of the issues before Java 11 is officially supported in Jenkins.
+
+In this blog post we will describe how to run with Java 11,
+and how to investigate compatibility issues and report them.
+
+=== Background
+
+As you probably remember,
+in June 2018 we had an link:/blog/2018/06/08/jenkins-java10-hackathon/[online hackathon] targeting the Java 10+ support in Jenkins.
+As a part of the hackathon,
+we provided the link:/blog/2018/06/17/running-jenkins-with-java10-11/[experimental support of Java 11].
+This event was a big success for us,
+because we were able to get Jenkins running with Java 10 and Java 11 early access,
+including major features like Jenkins Pipeline, Configuration as Code, BlueOcean, etc.
+It gave us confidence that we can provide the Java 11 support in Jenkins without major breaking changes.
+
+After the hackathon, link:https://github.com/oleg-nenashev/[Oleg Nenashev] has created
+jep:211[JEP-211: "Java 10+ support in Jenkins"].
+After the Java 11 GA release on Sep 25 (and Java 10 End of Life on the same date),
+the JEP has been adjusted to target Java 11 only.
+link:/sigs/platform[Platform Special Interest Group]
+has been also founded to coordinate the Java 11 support work
+and other platform support efforts (packaging, operating system support, etc.).
+A group of contributors continued working on Java 10+ support,
+mostly focusing on upstreaming functional patches and
+enabling Java 11 in Jenkins development tools.
+See the link:/sigs/platform/#meetings[Platform SIG meeting notes] for detailed status updates.
+
+// TODO: Add some photos from hackfests
```

Comment:
Is this something you want to add before publishing? or is it just a "nice to have"?

Reply:
removed for now